.github/workflows/README.md

```diff
@@ -0,0 +1,126 @@
+
+#### Version
+The file `<package_name>/__init__.py` must contain the following line:
+
+```python
+__version__ = "<version to release, e.g. 1.2.3>"
+```
+
+#### Install scripts
+The workflow looks for platform-specific subdirectories `linux`, `macos`, `windows` in the
+`release` directory of the calling repository. If the directory exists, an artifact is build for that platform.
+The subdirectories must contain the following scripts:
+
+```
+release/<platform>/build_installer_<platform>.sh
+release/<platform>/build_package_<platform>.sh
+release/<platform>/build_gui_<platform>.sh (optional)
+```
+or `.ps1` if platform is `windows`.
+
+
+
+### Installation
+#### 'Create release' workflow
+1. Create a new github action `create_release.yml` in `.github/workflows` like this:
+```yaml
+# Create a draft release and build and upload all installers to it.
+name: Create Draft Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_to_release:
+        description: 'Enter commit hash to release (example: ef4037cb571f99cb4919b520fde7174972aae473)'
+        type: string
+        required: true
+      tag_to_release:
+        description: 'Enter tag of the new release to create (example: v1.5.5). The code version needs to be bumped already tom atch the tag.'
+        type: string
+        required: true
+
+jobs:
+  create-release:
+    secrets: inherit
+    uses: MannLabs/alphashared/.github/workflows/create_release.yml@v1
+    with:
+      # see the documentation of the action for more information on the parameters
+      package_name: <Name of package, e.g. "alphadia", "peptdeep", ..>
+      commit_to_release: ${{ inputs.commit_to_release }}
+      tag_to_release: ${{ inputs.tag_to_release }}
+      # optional parameters
+      build_nodejs_ui: true
+      test_app: true
+      python_version: 3.9
+
+```
+
+#### 'Publish on pypi' workflow
+1. Create a new github action `publish_on_pypi.yml` in `.github/workflows` like this:
+```yaml
+# Publish and test release on Test-PyPI and PyPI.
+name: Publish on PyPi
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag_to_release:
+        description: 'Enter tag to release (example: v1.5.5). A tag with the same name must exist in the repository.'
+        type: string
+        required: true
+
+jobs:
+  publish_on_pypi:
+    uses: MannLabs/alphashared/.github/workflows/publish_on_pypi.yml@v1
+    secrets:
+      test_pypi_api_token: ${{ secrets.TEST_PYPI_API_TOKEN }}
+      pypi_api_token: ${{ secrets.PYPI_API_TOKEN }}
+    with:
+      # see the documentation of the action for more information on the parameters
+      package_name: <Name of package, e.g. "alphadia", "peptdeep", ..>
+      tag_to_release: ${{ inputs.tag_to_release }}
+      # optional parameters:
+      python_version: 3.9
+      use_pyproject_toml: false
+```
+
+
+## Development
+If you need to change the basic workflow, after merging to `main`, create a new `TAG`
```

Comment:
the basic workflow of what? Could you give an example of why one would need to change this

Reply:
should be clearer now